### PR TITLE
Adds a new captain id fire cabinet to captains office with functionality.

### PR DESCRIPTION
**Pull request by JamieD1**
Adds fire cabinet with the spare id instead of breaking open the locker every round.

https://i.imgur.com/Ndjh94e.gif

#### Changelog

:cl:  
rscadd: Added new spare id locker in captains room
rscdel: Removed spare from captains locker
/:cl:

### CODE REVIEW

File: _maps/map_files/YogStation/YogStation.dmm [hunk 1]
```diff
@@ -42784,6 +42774,16 @@
 	},
 /turf/open/floor/plating,
 /area/hallway/secondary/exit)
+"hSy" = (
+/obj/structure/chair/comfy/brown{
+	dir = 4
+	},
+/obj/machinery/camera/motion{
+	c_tag = "Captains Office";
```
Comment (patpol4) [inline _maps/map_files/YogStation/YogStation.dmm]: Captain's Quarters

**Comment by Bibby0110** [comment]
godawful code haha

File: _maps/map_files/YogStation/YogStation.dmm [hunk 2]
```diff
@@ -50647,6 +50647,14 @@
 	},
 /turf/open/space/basic,
 /area/space)
+"ofG" = (
+/obj/structure/fireaxecabinet/spare{
+	pixel_x = 2;
+	pixel_y = 30;
```
Comment (patpol4) [inline _maps/map_files/YogStation/YogStation.dmm]: only pixel_y = 32

File: _maps/map_files/YogStation/YogStation.dmm [hunk 3]
```diff
@@ -50647,6 +50647,14 @@
...
+	req_access = list(20)
```
Comment (patpol4) [inline _maps/map_files/YogStation/YogStation.dmm]: pretty sure you have to do req_access_txt = "20"

File: _maps/map_files/YogStation/YogStation.dmm [hunk 4]
```diff
@@ -55202,6 +55210,14 @@
 /obj/machinery/portable_atmospherics/canister/oxygen,
 /turf/open/floor/plasteel,
 /area/medical/storage)
+"rLs" = (
+/obj/structure/dresser,
+/obj/structure/dresser,
```
Comment (patpol4) [inline _maps/map_files/YogStation/YogStation.dmm]: double dresser hmmmmmmm

**Review by athena413 — CHANGES_REQUESTED** [review]
This PR only applies to box, meaning that spare IDs will only exist on Boxstation, and not:

Eclipsestation
Metastation
IceBox
Omegastation
Deltastation

**Comment by JamieD1** [comment]
> This PR only applies to box, meaning that spare IDs will only exist on Boxstation, and not:
> 
> Eclipsestation
> Metastation
> IceBox
> Omegastation
> Deltastation

Done

**Review by athena413 — APPROVED** [review]
Looks good, I just think that this intersecting the air alarm on metastation looks real fucking weird
image

**Review by patpol4 — CHANGES_REQUESTED** [review]
What Athena said and my reviews from box carried over to the other maps if it applies

**Comment by JamieD1** [comment]
I hate you weje

**Pull request merged**